pkgs/top-level/all-packages.nix

```diff
@@ -2774,6 +2774,8 @@ let
 
   pytrainer = callPackage ../applications/misc/pytrainer { };
 
+  remarshal = (callPackage ../development/tools/remarshal { }).bin;
```

Comment:
Missing `// { outputs = [ "bin" ]; }` otherwise environment installations will have a bunch of unneeded bloat.